src/mrack/host.py

```diff
@@ -84,25 +84,24 @@ def __init__(
 
     def __str__(self):
         """Return string representation of host."""
-        net_str = " ".join(self._ips)
+        net_str = " ".join(self._ip_addrs)
 
         out = (
-            f"{self._status} {self._id} {self._name} {net_str} {self._username} "
+            f"{self._status} {self._host_id} {self._name} {net_str} {self._username} "
             f"{self._password}"
         )
 
         if self._error:
-            o = [out, "Error:", object2json(self._error)]
-            out = "\n".join(o)
+            out = "\n".join([out, "Error:", object2json(self._error)])
         return out
 
     def to_json(self):
         """Transform object into representation which is acceptable by `json.dump`."""
         return {
             "provider": self._provider.name,
-            "id": self._id,
+            "host_id": self._host_id,
```

Comment:
This is backwards incompatible change. Are we OK with it?

Reply:
@netoarmando what do you say

Reply:
I'm okay with this, I like to use verbose variables for things like `id`, `type`, `instance`. 

But based on another comment from Petr, the issue here is the number of characters, right? So, do we need to change the key as well? Would that make this backwards compatible?

Reply:
Issue is that loading/saving mrack db which was created before this change will not work. But I guess it is acceptable given that we did not release a 1.0 yet.